infra/scripts/rollback.sh

```
@@ -0,0 +1,75 @@
+  if [[ $confirm != [yY] ]]; then
+    echo "Aborted."
+    exit 0
+  fi
+fi
+
+echo "=== Multi-Cloud Rollback: OpenLegislation ($ENV) ==="
+echo "Provider: $PROVIDER"
+
+# Load env vars
+if [ -f "infra/terraform/terraform.tfvars.$ENV" ]; then
+  export $(grep -v '^#' infra/terraform/terraform.tfvars.$ENV | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)